.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax
 
-* @ONSdigital/dissemination-open-sauce
+* @ONSdigital/dissemination-archivists
```

Comment:
There isn't such a team…
https://github.com/orgs/ONSdigital/teams/dissemination/teams?query=dissemination

Reply:
@lindenmckenzie has fixed the Team's settings so it is now valid.